src/test/java/seedu/guest/AppParametersTest.java

```diff
@@ -37,6 +39,15 @@ public void parse_invalidConfigPath_success() {
         assertEquals(expected, AppParameters.parse(parametersStub));
     }
 
+    @Test
+    public void equals() {
+        // same object -> returns true
+        assertTrue(expected.equals(expected));
```

Comment:
This practice is used by the previous tests as well and it may not equals true if the equals function is not programmed correctly hence we need to test it.